src/coqLsp/coqLspTypes.ts

```diff
@@ -156,3 +156,13 @@ export class CoqLspError extends Error {
         this.name = "CoqLspError";
     }
 }
+
+export class CoqLspStartupError extends Error {
+    constructor(
+        message: string,
+        public readonly path: string
```

Comment:
`public` is redundant here (and according to our unwritten code style rules, we usually omit it)